apps/studio/src/features/mail/service.ts

```diff
@@ -0,0 +1,40 @@
+import type { InvitationEmailTemplateData } from "./templates"
+import { createBaseLogger } from "~/lib/logger"
+import { isValidEmail } from "~/utils/email"
+import { sendMail } from "../../lib/mail"
+import { templates } from "./templates/templates"
+
+const logger = createBaseLogger({ path: "lib/mail/service" })
+
+export async function sendInvitation(
```

Comment:
[FYI] This is created on teh frontend - you can check the dockerfile to confirm this, not a